PWGHF/vertexingHF/AliAnalysisTaskSEXic0Semileptonic.h

```diff
@@ -152,7 +161,7 @@ class AliAnalysisTaskSEXic0Semileptonic : public AliAnalysisTaskSE
     //Double_t fNClustersTPCMin = 70;
     Double_t fNClustersITSMin = 2;
     Double_t fNXiCrossedRowsMin = 70;
-    Double_t fNXiCrossedRowsOverFindalbeRatioMin = 0.6;
+    Double_t fNXiCrossedRowsOverFindalbeRatioMin = 0.77;
 
     ClassDef(AliAnalysisTaskSEXic0Semileptonic, 1);
```

Comment:
Since you added three data members you should increase by 1 the ClassDef (from 1 to 2)

Reply:
Thanks Fabrizio! I modified it